README.md

```diff
@@ -23,11 +23,16 @@ Java, either on the system, or better, via some asdf-java
 Plugin:
 
 ```shell
-asdf plugin add djinni
-# or
-asdf plugin add https://github.com/cross-language-cpp/asdf-djinni.git
+asdf plugin add djinni https://github.com/cross-language-cpp/asdf-djinni.git
 ```
 
+or, once this plugin will be in the [asdf-plugin repo](https://github.com/asdf-vm/asdf-plugins)
+
+```shell
+asdf plugin add djinni https://github.com/cross-language-cpp/asdf-djinni.git
```

Comment:
Why is this the same command, even if the plugin will be an the asdf-plugin repo? Don't know anything about asdf but my understandig is that is should be available with a command like `asdf plugin add djinni` or something like that?

Reply:
good catch! 
yes, copy and past error 

I like to work with you very much!!